server/mscalendar/availability_test.go

```diff
@@ -293,6 +295,30 @@ func TestReminders(t *testing.T) {
 			numReminders:   2,
 			shouldLogError: false,
 		},
+		"Remote event linked to channel in the range for the reminder. DM and channel reminders should occur.": {
+			remoteEvents: []*remote.Event{
+				{ID: "event_id", ICalUID: "event_id", Start: remote.NewDateTime(time.Now().Add(7*time.Minute).UTC(), "UTC"), End: remote.NewDateTime(time.Now().Add(45*time.Minute).UTC(), "UTC")},
+			},
+			eventMetadata: map[string]*store.EventMetadata{
+				"event_id": {
+					LinkedChannels: map[string]struct{}{"channel_id": {}},
```

Comment:
```suggestion
					LinkedChannels: map[string]struct{}{"some_channel_id": {}},
```

Just to differentiate from `channel_id` potentially being used as a namespace